Backup: add note to pick a time when all locations are available (#37231)
home-assistant/home-assistant.github.io: source/_includes/common-tasks/backups.md

   - It is recommended to back up **Daily**, but you can also choose to back up on specific days.
   - Define the time:
     - **System optimal** sets a time in a predefined time window as shown in the UI.
     - **Custom** lets you pick the time when you want the backup to start.

>      - Make sure you pick a time when all your backup locations are up and running and available. Otherwise, the backup will fail for the location that is not available.

Reviewer: ```suggestion
     - Make sure you pick a time when all your backup locations are up and running and available. Otherwise, the backup will fail for locations which are not available.
```

1. Go to {% my supervisor_backups title="**Settings** > **System** > **Backups**" %} and under **Automatic backups**, select **Configure automatic backups**.
2. Under **Locations**, use the toggle to enable all the backup locations you want to use.
   - If you don't see Home Assistant Cloud in the list, you are not [logged in](https://www.nabucasa.com/config/).

>    - If you want to back up to your NAS (such as [Synology](/integrations/synology_dsm/#backup-location)) or a cloud provider (such as [GoogleDrive](/integrations/google_drive/) or [Microsoft OneDrive](/integrations/onedrive/)), check their integration documentation for specific instructions on setting up a Home Assistant backup.

Reviewer: ```suggestion
   - If you want to back up to your NAS (such as [Synology](/integrations/synology_dsm/#backup-location)) or a cloud provider (such as [Google Drive](/integrations/google_drive/) or [Microsoft OneDrive](/integrations/onedrive/)), check their integration documentation for specific instructions on setting up a Home Assistant backup.
```
